pages/common/luac.md

```diff
@@ -0,0 +1,11 @@
+# luac
+
+> Lua byte code compiler.
+
+- Compile a lua source code to a lua byte code:
```

Comment:
Perhaps ...`Lua source file to Lua bytecode would read better?

Reply:
I agree. I'll change it.